src/main/java/ox/app/states/PreEndState.java

```diff
@@ -0,0 +1,107 @@
+package ox.app.states;
+
+import ox.app.game.ScoreBoard;
+import ox.app.game.Board;
+import ox.app.game.Player;
+import ox.app.utility.RoundBuffer;
+
+import java.util.function.Consumer;
+
+public class PreEndState implements GameState {
+
+    private final Board board;
+    private final RoundBuffer playerBuffer;
```

Comment:
RoundBuffer looks for me like Players class. It is handling stuff purelly connected with players, not with rounds

Reply:
Round means rounded in that case :P